packages/devtools_app/lib/src/screens/memory/shared/primitives/instance_set_view.dart

```diff
@@ -38,41 +44,47 @@ class InstanceSetView extends StatelessWidget {
         if (showMenu)
           ContextMenuButton(
             style: textStyle,
-            menu: _menu(),
+            menu: _menu(sampleObtainer!),
           ),
         if (!showMenu) const SizedBox(width: ContextMenuButton.width),
       ],
     );
   }
 }
 
-class _MenuForSubset extends StatelessWidget {
-  const _MenuForSubset(this.menuText);
+class _StoreAsVariableMenu extends StatelessWidget {
+  const _StoreAsVariableMenu(this.menuText, this.sampleObtainer);
 
   final String menuText;
+  final ClassSampler sampleObtainer;
 
   @override
   Widget build(BuildContext context) {
     return SubmenuButton(
       menuChildren: <Widget>[
         MenuItemButton(
-          onPressed: () => {},
-          child: const Text('Fields'),
+          onPressed: sampleObtainer.oneVariableToConsole,
+          child: const Text('One instance'),
         ),
         const MenuItemButton(
-          child: Text('Outgoing references'),
+          child: Text('First 20 instances'),
         ),
         const MenuItemButton(
-          child: Text('Incoming references'),
+          child: Text('All instances'),
         ),
       ],
       child: Text(menuText),
     );
   }
 }
 
-List<Widget> _menu() => [
-      const _MenuForSubset('Store one instance as a console variable'),
-      const _MenuForSubset('Store first 100 instances as a console variable'),
-      const _MenuForSubset('Store all instances as a console variable'),
+List<Widget> _menu(ClassSampler sampleObtainer) => [
+      _StoreAsVariableMenu(
+        'Store as a console variable',
+        sampleObtainer,
+      ),
+      MenuItemButton(
+        onPressed: sampleObtainer.instanceGraphToConsole,
+        child: const Text('Browse an instance references in console'),
```

Comment:
This wording could use a little work. Maybe "Browser references for a single instance"

Reply:
Made it 'Browse references for a single instance in console'

I want the word 'console', because without it user will not pay attention to console and will miss what is going on.